polaris-react/src/components/ResourceList/ResourceList.tsx

```diff
@@ -759,11 +759,11 @@ export function ResourceList<TItemType extends ResourceListItemData>({
 
   return (
     <ResourceListContext.Provider value={context}>
-      <EventListener event="resize" handler={handleResize} />
       <div className={resourceListWrapperClasses} ref={tableMeasurerRef}>
         {filterControlMarkup}
         {headerMarkup}
         {listMarkup}
+        {bulkActionsMarkup}
```

Comment:
Thanks for this @vladovidiu, this would indeed an elegant solution to the [issue reported](https://github.com/Shopify/polaris-summer-editions/issues/667)
```suggestion
        {bulkActionsMarkup}
        {listMarkup}
```
Minor nit though, we should flip the render order of listMarkup and bulkActionsMarkup, so as to preserve keyboard navigation order as per the [root PR](https://github.com/Shopify/polaris/pull/7833)